bitnami/jaeger/templates/_helpers.tpl

```diff
@@ -0,0 +1,185 @@
+    {{ default (include "jaeger.collector.fullname" .) .Values.collector.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.collector.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use for the agent
+*/}}
+{{- define "jaeger.agent.serviceAccountName" -}}
+{{- if .Values.agent.serviceAccount.create -}}
+    {{ default (include "jaeger.agent.fullname" .) .Values.agent.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.agent.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use for the query
+*/}}
+{{- define "jaeger.query.serviceAccountName" -}}
+{{- if .Values.query.serviceAccount.create -}}
+    {{ default (include "jaeger.query.fullname" .) .Values.query.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.query.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create the name of the collector deployment
+*/}}
+{{- define "jaeger.collector.fullname" -}}
+    {{ printf "%s-collector" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Create the name of the collector deployment. This name includes 2 hyphens due to
+an issue about env vars collision with the chart name when the release name is set to just 'jaeger'
+ref. https://github.com/jaegertracing/jaeger-operator/issues/1158
+*/}}
+{{- define "jaeger.agent.fullname" -}}
+    {{ printf "%s--agent" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Create the cassandra secret name
+*/}}
+{{- define "jaeger.cassandra.secretName" -}}
+    {{- if not .Values.cassandra.enabled -}}
+        {{- .Values.externalDatabase.existingSecret -}}
+    {{- else -}}
+        {{- printf "%s-cassandra" (include "common.names.fullname" .) -}}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Create the cassandra secret key
+*/}}
+{{- define "jaeger.cassandra.secretKey" -}}
+    {{- if not .Values.cassandra.enabled -}}
+        cassandra-password
+    {{- else -}}
+        {{- .Values.externalDatabase.existingSecretPasswordKey -}}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Create the cassandra user
+*/}}
+{{- define "jaeger.cassandra.user" -}}
+    {{- if not .Values.cassandra.enabled -}}
+        {{- .Values.externalDatabase.dbUser.user | quote -}}
+    {{- else -}}
+        {{- .Values.cassandra.dbUser.user | quote -}}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Create the cassandra password
+*/}}
+{{- define "jaeger.cassandra.password" -}}
+    {{- if not .Values.cassandra.enabled -}}
+        {{- .Values.externalDatabase.dbUser.password | quote -}}
+    {{- else -}}
+        {{ .Values.cassandra.dbUser.password | quote -}}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Create the cassandra host
+*/}}
+{{- define "jaeger.cassandra.host" -}}
+    {{- if not .Values.cassandra.enabled -}}
+        {{- .Values.externalDatabase.host | quote -}}
+    {{- else -}}
+        {{- printf "%s-cassandra" (include "common.names.fullname" .) -}}
```

Comment:
Use this helper

```
{{- include "common.names.dependency.fullname" (dict "chartName" "cassandra" "chartValues" .Values.cassandra "context" $) -}}
```

This takes into account some subchart values like nameOverride or fullnameOverride that you are not currently taking into account

Reply:
Done [0108b7a](https://github.com/bitnami/charts/pull/13480/commits/0108b7aedc639e16498c48d9b7c158c01fb3328f)